libmscore/tempotext.cpp

```diff
@@ -190,6 +190,7 @@ void TempoText::textChanged()
                         }
                   }
             }
+      score()->fixTicks();
```

Comment:
I moved score()->fixTicks(); just before score()->setPlaylistDirty();
To handle the case where inspector changes tempo text, I want to add the line:
**score->fixTicks();**
just before the line:
https://github.com/musescore/MuseScore/blob/master/mscore/inspector/inspectorBase.cpp#L385
but score->fixTicks(); should execute here only in the case when 'this' is specifically of type Ms::InspectorTempoText and not just any Ms::InspectorBase. Can I use a dynamic_cast to check if this is true (like an instanceof check)? Its usually not thought of as good coding practice to use a dynamic_cast, if I am not wrong, but I don't see another way.